airflow/utils/db.py

```diff
@@ -117,7 +117,7 @@ class MappedClassProtocol(Protocol):
     "2.8.0": "10b52ebd31f7",
     "2.8.1": "88344c1d9134",
     "2.9.0": "1949afb29106",
-    "2.9.2": "0fd0c178cbe8",
+    "2.9.2": "686269002441",
```

Comment:
this should only change on the version right?

Reply:
This now points to the downstram of this revision (`down_revision = "686269002441"`) which is the last revision in 2.9.2

Reply:
yes, true what Hussein explained.